.travis.yml

```diff
@@ -4,23 +4,18 @@ php:
   - 7.1
   - 7.2
   - 7.3
-  - 7.4snapshot
-  - nightly
-
-matrix:
-  allow_failures:
-    - php: 7.4snapshot
-    - php: nightly
+  - 7.4
+  - 8.0
 
 before_script:
   # Make sure the locales used in the tests are installed
-  - composer install
+  - composer update --ignore-platform-req=php
```

Comment:
I admin I was lazy and didn't bothered setting `--ignore-platform-req=php` only on PHP 8 cause of the state of this library, shame on me. Ping me if you want me to be more precise